src/lib/Bond.ts

```diff
@@ -15,6 +15,7 @@ export enum NetworkID {
   ArbitrumTestnet = 421611,
   AvalancheTestnet = 43113,
   Avalanche = 43114,
+  Localhost = 8545,
```

Comment:
1337 actually

8545 is the port for RPC URL

Reply:
@ivelin I left a request here. Seems fine otherwise.

Reply:
@0xJem thank you. Will make the change.

Reply:
@0xJem done. Please take another look when you can. I also merged with new commits in `develop`.